### PR TITLE
chore: update package-lock on next

### PR DESCRIPTION
**Pull request by apaleslimghost**
i have no idea

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1087,6 +1086,746 @@
         "npm": "7.x || 8.x || 9.x"
       }
     },
+    "core/sandbox/node_modules/@dotcom-tool-kit/backend-heroku-app": {
```
Comment (ivomurrell) [inline package-lock.json:1089]: **issue**: why has the sandbox returned 😨

**Comment by joelcarr** [comment]
There are changes in here for `core/sandbox/node_modules/*`. It looks like this branch does not have the sandbox like with `main` so maybe it is cached in some way?

File: package-lock.json (hunk 1 above)
Comment (apaleslimghost) [reply package-lock.json:1089]: ohhhhhhh. the sandbox is still gitignored, but i still have a `package.json` in that folder which is being picked up in the install. that's annoying. i can't think of a way to skip that without removing it from the workspace which kind of defeats the point of having it gitignored in the monorepo? we ignore these changes though

Comment (ivomurrell) [reply package-lock.json:1089]: could we exclude it in the `workspaces` value in the root `package.json`?

Comment (apaleslimghost) [reply package-lock.json:1089]: i _think_ that would prevent it from picking up other workspace dependencies. we could use file version specifiers for those though?

**Pull request closed without merging**